src/OptimalTransport.jl

```diff
@@ -17,10 +17,13 @@ export emd, emd2
 export sinkhorn_stabilized, sinkhorn_stabilized_epsscaling, sinkhorn_barycenter
 export sinkhorn_unbalanced, sinkhorn_unbalanced2
 export quadreg
-export ot_cost, ot_plan
+export ot_cost, ot_plan, wasserstein, squared2wasserstein
 
```

Comment:
Why do you think a `squared2wasserstein` is necessary? Why not only `wasserstein`? The `ot_cost` would already calculate the `squared` case if the user wished so, or he could just square the output of the `wasserstein

Reply:
The squared version is useful when working with e.g. normal distributions. And it is quite expensive to first take the square root and then square the result again. Therefore also e.g. Distances contains a dedicated `SqEuclidean` distance and Distributions a `pdfL2squared`.